### DISCUSSION TITLE
patchomator jamf pro execution script.sh

### DISCUSSION THREAD

**Discussion by marckerr** [Mac-Nerd/patchomator#25] [General]
In the script "patchomator jamf pro execution script.sh" I'm not sure that the ( ) are necessary for the paths. Shellcheck using /bin/sh or bash (they don't like zsh yet) thinks an array is being used for those variables which causes other Shellcheck notices.
I'm using the following and the script appears to work fine.
InstallomatorPath="/usr/local/Installomator/Installomator.sh"
PatchomatorPath="/usr/local/Installomator/patchomator.sh"

**Comment by Mac-Nerd** [comment]
Thanks for the edit. 
Fixed in b42a6ba
